src/pages/main/NotFound.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { routes } from '../../config'
+
+import './NotFound.css'
+
+function NotFound() {
+  return (
+    <main className="NotFound__main">
+      <h2 className="NotFound__section-title">404 Page Not Found</h2>
+      <p>Sorry! We didn't find anything interesting for you here. Please, click the button below to go home.</p>
+      <a href={routes.home} className="btn btn-outline-primary">Go to home page</a>
```

Comment:
Please, use the `<Button />` component here. You can see an example on the Home page with **View places** button. 😊

Reply:
Ok, thanks :) Stupid mistake, I fixed it.

I think, we don't need manually set aria-label like in Home page.
This tag is needed when the content of the element does not allow you to understand the meaning. For example, in icon fonts. That's why I decide to not include aria-label here.

![image](https://user-images.githubusercontent.com/26366132/66812488-a614ca00-ef3b-11e9-8d00-e99feb034128.png)

Reply:
Maybe, it can be removed from `/src/pages/main/places/placesGrid/PlaceCard.js` and `src/pages/main/Home.js`. Availability will not suffer. It's up for you.

Reply:
Okay! 😊